test/e2e_test_secrets.sh

```diff
@@ -137,7 +137,7 @@ cat /dev/urandom | head -n 10 | base64 > randomblob
 
 # upload blob and sign it
 dgst=$(./cosign upload blob -f randomblob ${blobimg})
-./cosign sign --key ${signing_key} ${dgst}
+./cosign sign --key ${signing_key} --tlog-upload=true ${dgst}
```

Comment:
Unless you want to target the production rekor, which I don’t think we need to for e2e cosign tests,  this should be tlog-upload=false (true is the default), and for verify, you need insecure-skip-tlog-verify=true

Reply:
Oh hm, if these are working, this is actually fine. You can remove tlog-upload though since it’s the default. 

There’s a TODO below for adding tlog tests you can remove. 

I’d also recommend adding another test here where you do remove the upload, just to make sure that’s working as expected.

Reply:
@haydentherapper added a test let me know if that is good

Reply:
Looks good!